aeolus/messages.h

```diff
@@ -25,34 +24,33 @@
 #include "addsynth.h"
 #include "global.h"
 
-
-class ITC_mesg {
-   public:
-      ITC_mesg() {}
-      ITC_mesg(int) {};
-      };
+class ITC_mesg
+{
+public:
+    ITC_mesg() {}
+    ITC_mesg(int) {}
+};
 
 enum
 {
-    FM_SLAVE =  8,
-    FM_IFACE =  9,
+    FM_SLAVE = 8,
+    FM_IFACE = 9,
     FM_MODEL = 10,
     FM_IMIDI = 11,
     FM_AUDIO = 12,
     FM_TXTIP = 13,
-    TO_SLAVE =  8,
-    TO_IFACE =  9,
+    TO_SLAVE = 8,
+    TO_IFACE = 9,
```

Comment:
same here and probably in many other places

Reply:
All those manual formatting tweaks cannot be correctly processed by any script, because we rely on a human eye's interpretation in this situation, unfortunately.

Reply:
Doesn't make this issue any better. Porobably better not touch this with automated tools then at all and do not merge multiple spaces into into just one

Reply:
Worth truing, indeed. @igorkorsukov can we configure the script to not touch multiple spaces? What are the drawbacks of such configuration?

Reply:
Better indeed to leave this kind of formatting untouched. <code>enum class ElementFlag</code> in <code>libmscore/element.h</code> shows a very bad example of the reformatting.

Reply:
Added ignore alignment when assigning